homeassistant/components/renson/time.py

```diff
@@ -0,0 +1,102 @@
+"""Renson ventilation unit time."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime, time
+
+from _collections_abc import Callable
+from renson_endura_delta.field_enum import DAYTIME_FIELD, NIGHTTIME_FIELD, FieldEnum
+from renson_endura_delta.renson import RensonVentilation
+
+from homeassistant.components.time import TimeEntity, TimeEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import RensonCoordinator, RensonData
+from .const import DOMAIN
+from .entity import RensonEntity
+
+
+@dataclass(kw_only=True, frozen=True)
+class RensonTimeEntityDescription(TimeEntityDescription):
+    """Class describing Renson time entity."""
+
+    action_fn: Callable[[RensonVentilation, str], None]
+    field: FieldEnum
+
+
+ENTITY_DESCRIPTIONS: tuple[RensonTimeEntityDescription, ...] = (
+    RensonTimeEntityDescription(
+        key="day_time",
+        entity_category=EntityCategory.CONFIG,
+        translation_key="day_time",
+        action_fn=lambda api, time: api.set_day_time(time),
+        field=DAYTIME_FIELD,
+        has_entity_name=True,
+    ),
+    RensonTimeEntityDescription(
+        key="night_time",
+        translation_key="night_time",
+        entity_category=EntityCategory.CONFIG,
+        action_fn=lambda api, time: api.set_night_time(time),
+        field=NIGHTTIME_FIELD,
+        has_entity_name=True,
+    ),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Renson time platform."""
+
+    data: RensonData = hass.data[DOMAIN][config_entry.entry_id]
+
+    entities = [
+        RensonTime(description, data.api, data.coordinator)
+        for description in ENTITY_DESCRIPTIONS
+    ]
+
+    async_add_entities(entities, True)
```

Comment:
Is `True` here really needed? You already do a `async_config_entry_first_refresh` in `__init__.py` so you should have data when adding entities